Marlin/Configuration.h

```diff
@@ -1564,4 +1564,59 @@
   //#define FILAMENT_LCD_DISPLAY
 #endif
 
+//===========================================================================
+//============================ I2C Encoder Settings =========================
+//===========================================================================
+
+
+//#define I2C_ENCODERS_ENABLED
```

Comment:
Maybe I2C_POSITION_ENCODERS?  This way it's clearer what it is everywhere across the codebase.